### PR TITLE
Merge 11.6 to master

### PR DESCRIPTION
**Pull request by slclinica**

### CODE REVIEW

File: src/main/java/org/akaza/openclinica/config/ScheduledJobConfig.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        logger.info("In postProcessBeanFactory");
+
+        Set<String> schemas = studyDao.findAll().stream()
+                .map(study -> study.getSchemaName())
+                .collect(Collectors.toSet());
+
+        for (String schema : schemas) {
+            CoreResources.setRequestSchema(schema);
+            if (ocQrtzTriggersDAO.findAll().size() != 0) {
+                CoreResources.setRequestSchema("public");
```
Comment (svadla-oc) [inline src/main/java/org/akaza/openclinica/config/ScheduledJobConfig.java:47]: why are you switching the schema back to public?

File: src/main/java/core/org/akaza/openclinica/core/LiquibaseConfig.java [hunk 2]
```diff
@@ -11,6 +11,7 @@
  * Created by yogi on 3/22/17.
  */
 @Configuration
+@ComponentScan("ScheduledJobConfig.class")
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/core/LiquibaseConfig.java:14]: why do we need this?

File: src/main/java/org/akaza/openclinica/config/ScheduledJobConfig.java (hunk 1 above)
Comment (slclinica) [reply src/main/java/org/akaza/openclinica/config/ScheduledJobConfig.java:47]: Sorry I added it while debugging but forgot to remove.

File: src/main/java/core/org/akaza/openclinica/core/LiquibaseConfig.java (hunk 2 above)
Comment (slclinica) [reply src/main/java/core/org/akaza/openclinica/core/LiquibaseConfig.java:14]: The dependsOn annotation doesn't work when it's on its own on a component, you have to set componentScan along with it.

**Comment by svadla-oc** [comment]
👍

**Pull request merged**